infra/base-images/base-builder/compile_python_fuzzer

```diff
@@ -15,6 +15,9 @@
 #
 ################################################################################
 
+# In order to enable PySecSan for a given module, set the environment
```

Comment:
Why not a commandline argument?

Reply:
No elaborate reason as such. The benefits of having it as an env variable is the current use of `compile_python_fuzzer` makes it easy to pass arbitrary arguments to `pyinstaller`, e.g. https://github.com/google/oss-fuzz/blob/dd66f15b2073c25e02070e60ba813d7dd6d61002/projects/django/build.sh#L26 -- I think adding pysecsan as a commandline argument may complicate the script more than adding an env variable.

Am happy to leave as is.